client/src/constants/validation.js

```diff
@@ -7,13 +7,15 @@ const healthRegions = [
   'Vancouver Coastal',
   'Vancouver Island',
   'Northern',
+  '',
```

Comment:
this is an extreme hack to work around wanting to use nullable + oneOf when the value is filled in